test/cli.js

```diff
@@ -69,7 +70,7 @@ test('don\'t display test title if there is only one anonymous test', function (
 
 	execCli(['fixture/es2015.js'], function (err, stdout, stderr) {
 		t.ifError(err);
-		t.is(stderr.trim(), '1 test passed');
+		t.true(stderr.trim().includes('1 test passed'));
```

Comment:
This functionality should be tested by using API (`api.js`), without executing AVA's CLI.